.lando/elasticsearch.yml

```diff
@@ -0,0 +1,27 @@
+http:
```

Comment:
Why do we need to specify these values? What part varies from the defaults?

Reply:
I've used [Lando's example config](https://github.com/lando/lando/blob/master/examples/elasticsearch/config/custom.yml) and added `cors` settings in addition to that. What's your suggestion, remove `elasticsearch.yml` altogether?

Reply:
I'll review the defaults and remove those.

Reply:
Ok, I'll remove `elasticsearch.yml` and refer to docs. These are default values:

```
lando ssh -s elasticsearch -u root -c "cat /opt/bitnami/elasticsearch/config/elasticsearch.yml"
http:
  port: 9200
path:
  data: /bitnami/elasticsearch/data
transport:
  tcp:
    port: 9300
network:
  host: |-
    172.20.0.3
    172.28.0.6
  publish_host: |-
    172.20.0.3
    172.28.0.6
  bind_host: 0.0.0.0
cluster:
  name: bespin
node:
  name: lando
  master: true
  data: true
  ingest: false
discovery:
```

Reply:
@floretan , resolved here: b809fe3.